client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1585,32 +1572,31 @@ def populate_load_placeholder(self, placeholder, ignore_repre_ids=None):
         loader_name = placeholder.data["loader"]
         loader_args = self.parse_loader_args(placeholder.data["loader_args"])
 
-        placeholder_representations = self._get_representations(placeholder)
-
-        filtered_representations = []
-        for representation in self._reduce_last_version_repre_entities(
-            placeholder_representations
-        ):
-            repre_id = representation["id"]
-            if repre_id not in ignore_repre_ids:
-                filtered_representations.append(representation)
+        placeholder_representations = [
+            repre_entity
+            for repre_entity in self._get_representations(placeholder)
+            if repre_entity["id"] not in ignore_repre_ids
+        ]
 
-        if not filtered_representations:
+        repre_load_contexts = get_representation_contexts(
+            self.project_name, placeholder_representations
+        )
+        filtered_repre_contexts = self._reduce_last_version_repre_entities(
+            repre_load_contexts
```

Comment:
`get_representation_contexts` returns a `dict` with the contexts by representation id.
```suggestion
            repre_load_contexts.values()
```

When fixing this line, the next error is:
```
# Traceback (most recent call last):
#   File "E:\dev\ayon-core\client\ayon_core\pipeline\workfile\workfile_template_builder.py", line 716, in populate_scene_placeholders
#     placeholder_plugin.populate_placeholder(placeholder)
#   File "E:\dev\ayon-core\client\ayon_core\hosts\maya\api\workfile_template_builder.py", line 246, in populate_placeholder
#     self.populate_load_placeholder(placeholder)
#   File "E:\dev\ayon-core\client\ayon_core\pipeline\workfile\workfile_template_builder.py", line 1586, in populate_load_placeholder
#     filtered_repre_contexts = self._reduce_last_version_repre_entities(
#   File "E:\dev\ayon-core\client\ayon_core\pipeline\workfile\workfile_template_builder.py", line 1544, in _reduce_last_version_repre_entities
#     version_mapping[version].append(repre_context)
```